src/components/views/voip/CallView.tsx

```diff
@@ -25,6 +25,8 @@ import RoomAvatar from "../avatars/RoomAvatar";
 import { CallState, CallType, MatrixCall, CallEvent } from 'matrix-js-sdk/src/webrtc/call';
 import classNames from 'classnames';
 import AccessibleButton from '../elements/AccessibleButton';
+import AccessibleTooltipButton from "../elements/AccessibleTooltipButton";
+import { ContextMenuTooltipButton } from "../../../accessibility/context_menu/ContextMenuTooltipButton";
 import {isOnlyCtrlOrCmdKeyEvent, Key} from '../../../Keyboard';
 import {alwaysAboveLeftOf, alwaysAboveRightOf, ChevronFace, ContextMenuButton} from '../../structures/ContextMenu';
```

Comment:
```suggestion
import {isOnlyCtrlOrCmdKeyEvent, Key} from '../../../Keyboard';
import {
    alwaysAboveLeftOf,
    alwaysAboveRightOf,
    ChevronFace,
    ContextMenuTooltipButton,
} from '../../structures/ContextMenu';
```

Reply:
It should be re-exported by ContextMenu.

https://github.com/matrix-org/matrix-react-sdk/blob/f156c2db15e7e1bef5fd5e444be5dec644aac18d/src/components/structures/ContextMenu.tsx#L506

In any case, this suggestion is to fix the lint caused by an unused import (ContextMenuButton)